src/types.js

```diff
@@ -152,14 +168,16 @@ export type OfferType = {
   createdDate: string
 }
 
-export type JsonLanguageType = { code: string, native_name: string }
+export type JsonLanguageType = { code: string, native_name: ?any, name: ?any }
```

Comment:
native_name should follow the camelCase convention

Reply:
This is the same as the api response and it was like this from before so I left it as it is.

Reply:
Underscores are fine in the Json types :+1: